src/Exception/JsonException.php

```diff
@@ -0,0 +1,30 @@
+<?php
+/**
+ * This file is part of the prooph/pdo-event-store.
+ * (c) 2016-2017 prooph software GmbH <contact@prooph.de>
+ * (c) 2016-2017 Sascha-Oliver Prolic <saschaprolic@googlemail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Prooph\EventStore\Pdo\Exception;
+
+use Prooph\EventStore\Exception\RuntimeException as EventStoreRuntimeException;
+
+class JsonException extends EventStoreRuntimeException implements PdoEventStoreException
```

Comment:
@codeliner your thoughts on this?

Reply:
I like it especially the method `whileDecode`